packages/renderer/src/render-media.ts

```diff
@@ -382,37 +386,40 @@ export const renderMedia = ({
 
 			const stitchStart = Date.now();
 			return Promise.all([
-				stitchFramesToVideo({
-					width: composition.width * (scale ?? 1),
-					height: composition.height * (scale ?? 1),
-					fps,
-					outputLocation: absoluteOutputLocation,
-					internalOptions: {
-						preEncodedFileLocation,
-						imageFormat,
+				stitchFramesToVideo(
```

Comment:
I forgot to say: `stitchFramesToVideo` is also a public API and should therefore not accept the `remotionRoot` parameter (list of public APIs is here: https://www.remotion.dev/docs/renderer)